### PR TITLE
feat: add endpoints for Addresses

### PR DESCRIPTION
**Pull request by zaibugemi**

### CODE REVIEW

**Comment by zaibugemi** [comment]
@ageddesi would you please review this PR when you have time?

File: modules/address/api/address-routes.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import getRandomAddresses from '../utils/getRandomAddress';
+import { getQtyFromRequest } from '../../../utils/route-utils';
+import { getCountryNameFromRequest } from '../../../utils/route-utils';
+
+module.exports = function(app : core.Express) {
+
+	// get a random address (by default, a UK address is returned)
+	app.get('/address', (req: Request, res: Response) => {
```
Comment (ageddesi) [inline modules/address/api/address-routes.ts:10]: Please can you add OpenAPI comments to each endpoint please.

File: modules/address/api/address-routes.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { Request, Response } from 'express';
```
Comment (ageddesi) [inline modules/address/api/address-routes.ts:1]: Can you add unit tests for the endpoints please

File: modules/address/utils/getRandomAddress.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { faker } from '@faker-js/faker';
```
Comment (ageddesi) [inline modules/address/utils/getRandomAddress.ts:1]: Can you add unit tests for this please

**Review by ageddesi — CHANGES_REQUESTED** [review]
Thank you @zaibugemi for the hard work. There is just a couple of extra bits we need before we can accept the code. 👍

**Comment by zaibugemi** [comment]
@ageddesi sure, let me work on your suggestions. All that work now needs to be done in a new PR, right, as this one is closed?

**Comment by zaibugemi** [comment]
@ageddesi also will you also please mark this PR for `hacktoberfest` when you accept this feature?

**Pull request closed without merging**